apps/site/pages/components/molecules/card.mdx

```diff
@@ -51,17 +51,17 @@ Card displays some information divided into a body and a header.
       iconName="PlusCircle"
       title="Card with Icon"
       style={{width: '300px'}}>
-      <h1> First Product </h1>
+      <p> First Product </p>
```

Comment:
question: Was this change related to accessibility?

Reply:
Yes, it was also related. My concern primarily was around [heading hierarchy](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). On pages where we have multiple cards, the `h1` tag should be reserved for the main heading of the page. Since heading structure depends on context, I think is better to not use an `h1` tag in this example. WTYT?

Reply:
Agree!